tools/assets/n64texconv/__init__.py

```diff
@@ -245,6 +245,13 @@ def reformat(self, fmt : int) -> Optional["N64Palette"]:
         _object_refcount.add_ref(pal)
         return deref(pal)
 
+    def resize(self, new_count : int) -> Optional["N64Palette"]:
+        pal = ln64texconv.n64texconv_palette_resize(byref(self), new_count)
+        if new_count > 256:
+            raise ValueError("The largest possible palette size is 256")
```

Comment:
wouldn't it make more sense to check new_count before calling n64texconv_palette_resize ?